src/goto-symex/goto_symex_state.cpp

```diff
@@ -277,7 +270,10 @@ ssa_exprt goto_symex_statet::rename_ssa(ssa_exprt ssa, const namespacet &ns)
     ssa = ssa_l0.get();
   }
   else if(level == L1)
-    set_l1_indices(ssa, ns);
+  {
+    const renamedt<ssa_exprt, L1> ssa_l1 = set_l1_indices(std::move(ssa), ns);
+    ssa = ssa_l1.get();
```

Comment:
Like for the level-0 case, I'd prefer `ssa = set_l1_indices(std::move(ssa), ns).get();`